### PR TITLE
EN-2254 EN-2255 categories mock API, redux state and tests

### PR DESCRIPTION
**Pull request by otarza**
1. https://jira.entando.org/browse/EN-2254
2. https://jira.entando.org/browse/EN-2255

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/state/digital-exchange/categories/actions.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export const setDECategories = digitalExchangeCategories => ({
+  type: SET_DE_CATEGORIES,
+  payload: {
+    digitalExchangeCategories,
+  },
+});
+
+export const fetchDECategories = () => dispatch => (
+  new Promise((resolve) => {
+    dispatch(toggleLoading('digital-exchange/categories'));
```
Comment (gidesan) [inline src/state/digital-exchange/categories/actions.js]: what about extracting `'digital-exchange/categories'` to a constant?

File: src/state/digital-exchange/categories/actions.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    getDECategories().then((response) => {
+      response.json().then((data) => {
+        if (response.ok) {
+          dispatch(setDECategories(data.payload));
+          dispatch(toggleLoading('digital-exchange/categories'));
```
Comment (gidesan) [inline src/state/digital-exchange/categories/actions.js]: can we move the `dispatch(toggleLoading('digital-exchange/categories'));` outside the if / else ?

**Comment by entando-service** [comment]
All is well

**Pull request merged**